### DISCUSSION TITLE
SeleniumBase doesn't use certain Xvfb display

### DISCUSSION THREAD

**Discussion by serg-sno** [seleniumbase/SeleniumBase#3143] [General]
Hello!
I think I didn’t explain the problem clearly enough [here](https://github.com/seleniumbase/SeleniumBase/discussions/3137), apologies for that. I don’t want SeleniumBase to fake the screen resolution. I’m working with a virtual machine using a virtual display through Xvfb. What I need is to create a virtual display with a resolution of 1920x1080, for example by running the command:
`Xvfb -ac :99 -screen 0 1920x1080x16`
and have SeleniumBase utilize it. Alternatively, it would be helpful if SeleniumBase could create an Xvfb display with a custom resolution. Currently, SeleniumBase sets up an Xvfb display with an unusual resolution of 1366x768, which might raise suspicion from the web service.
![1](https://github.com/user-attachments/assets/1fe389df-d73b-42c6-8ba1-f833207af7c7)

**Comment by mdmintz** [comment]
1366x768 is the standard display size on Linux servers, but I can add an option to customize it.

**Reply by serg-sno** [reply]
Thank you, @mdmintz! That would be great, as today real customers don't use a screen resolution of 1366x768, and using it in SeleniumBase might seem suspicious.

**Reply by mdmintz** [reply]
The 1366x768 resolution is actually quite common:
* https://www.reddit.com/r/The10thDentist/comments/j47xkd/1366x768_resolution_is_absolutely_fine/
* https://en.wikipedia.org/wiki/Display_resolution_standards#1366x768
* https://superuser.com/questions/946086/why-does-1366x768-resolution-exist

And for some Ubuntu laptops, it's the maximum resolution:
* https://ubuntu-mate.community/t/design-choice-or-bug-for-1366x768-screen/25606

I've been testing UC Mode for awhile on 1366x768 Ubuntu Linux: No issues with that.

**Reply by serg-sno** [reply]
A 1366x768 resolution might be sufficient for CLI, but nowadays, I don't know anyone who uses that resolution for web browsing on a laptop. Perhaps my experience isn't representative, but I still think it would be beneficial to have the option to customize it.

**Reply by mdmintz** [reply]
Yes, that option will be added in the next release.
